app/views/accounts/_bio.html.haml

```diff
@@ -5,17 +5,17 @@
     .account__header__fields
       - fields.each do |field|
         %dl
-          %dt.emojify{ title: field.name }= Formatter.instance.format_field(account, field.name, custom_emojify: true)
+          %dt.emojify{ title: field.name }= prerender_custom_emojis(h(field.name), account.emojis)
```

Comment:
Isn't it missing the linking part?
i.e., shouldn't it be `prerender_custom_emojis(html_aware_format(field.name, account.local?, with_rel_me: true, with_domains: true, multiline: false), account.emojis)`?

Reply:
No, field names were never supposed to get anything except custom emojis. In the actor serializer, the field names have never been passed through the formatter at all, for example.

Reply:
Hm alright. This *is* a behavior change, but the current behavior is pretty inconsistent. Fine by me.